pvm.py

```diff
@@ -36,29 +37,90 @@ def _init_logger():
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
 IS_FILE_SET = False
+NEXT_TIME = None
+
+'''
+UDP command example:
+/PVM 12:13:14
+/PVM <action_name> <action_value>
+
+The normal workflow is below,
+1. Receive the first command: /PVM 12:13:14
+	- parse the time
+	- plus 3s
+	- save the time into NEXT_TIME
+2. Receive the second command: /PVM file jellyfish.mp4
+	- Wait until the NEXT_TIME
+	- Execute the command
+	- Set NEXT_TIME = None
+'''
 
 def parse_commands(*args):
 	global media
 	global VIDEO_PATH
 	global IS_FILE_SET
+	global NEXT_TIME
+	# Parse the time send in UDP, 
+	# From /PVM 12 13 14 to 12:13:14
+	if len(args) == 4:
```

Comment:
this will not happen now, remember the message format is `/PVM HH MM SS <action_name> <action_value>`
you need to slice the first arguments here first and then pass the rest to be parsed below

Reply:
Done